yt/data_objects/construction_data_containers.py

```diff
@@ -666,6 +666,9 @@ def __init__(
         self.level = level
         self.left_edge = self._sanitize_edge(left_edge)
         self.ActiveDimensions = self._sanitize_dims(dims)
+        self.fix_nonperiodic = [False, False, False]
+        self.lo_buffer = np.array([0, 0, 0], dtype="int64")
+        self.hi_buffer = np.array([0, 0, 0], dtype="int64")
```

Comment:
From the names only, it's really not clear to me what these attribute are used for, especially `fix_periodicity` because it shares a name with a function you're importing.

Reply:
I agree, let me change this around and come up with something more clear